### ISSUE TITLE
Shortcuts as in Dreame App

### ISSUE THREAD

**Issue by robitsrl** [Tasshack/dreame-vacuum#493]
**Is your feature request related to a problem? Please describe.**
It would be nice to have a new feature like the dreame app where there are shortcuts. With shortcuts you can create on for example to first clean a room and then mop the same room or someting else.

**Describe the solution you'd like**
Make a script where i can have sequence of cleaning. Now if i create a script with multiple jobs only the forst job is been done
Probably it's already possibile but in the case i miss something.

**Describe alternatives you've considered**
I have tried to make a script when after the first job I wait until the robot goes back in the base before starting the new job but does not work 

**Additional context**

**Comment by robitsrl** [comment]
Hi,
but the Xiaomi app do not have shortcuts as the dream app so i do not have button.{vacuum}_shortcut_{id}

Maybee now is possibile to use DreamApp account with homeassistant ?

**Comment by robitsrl** [comment]
Ok, with dreame account will works also L10 Prime or only the same model of xiaomi account ?
